### DISCUSSION TITLE
Is nominatim.openstreetmap.org already updated?

### DISCUSSION THREAD

**Discussion by otbutz** [osm-search/Nominatim#2507] [Q&A] [answered]
According to the [blog post](https://nominatim.org/2021/11/03/release-40.html) the OSM instance should already use the new tokenizer:

> In fact, the legacy tokenizer is still the default choice for a custom import. A database using the ICU tokenizer is already deployed at the reference site at https://nominatim.openstreetmap.org, where you can try it out.

But the server status indicates that it's still on 3.7.0-2?

https://nominatim.openstreetmap.org/ui/status.html

**Comment by lonvia** [comment, marked as answer]
Yes, it is up to date except for the last commit which changes the version number.
